File: CODECHEF/JULY20B/CRDGAME.cpp
#include <iostream>
#include <bits/stdc++.h>
using namespace std;
long long int Sum(long long int a)
{
	long long int s =0;
	while(a!=0)
	{
		s = s + a%10;
		a = a/10;
	}
	return s;
}
int main()
{
	int t;
	cin>>t;
	while(t--)
	{
		int n;
		cin>>n;
		long long int pointc = 0,pointm=0,pc,pm;
		long long int c,m;
		while(n--)
		{
			cin>>c>>m;
			pc = Sum(c);
			pm= Sum(m);
			if(pc == pm)
			{
				pointc++;
				pointm++;
			}
			else if(pc > pm)
				pointc++;
			else
				pointm++;
		
		}
		
		if(pointc == pointm)
			cout<<2<<" "<<pointc<< endl;
		else if(pointc > pointm)
			cout<<0<<" "<<pointc<<endl;
		else
			cout<<1<<" "<<pointm<<endl;
	}
	return 0;
}

File: CODECHEF/JULY20B/CRDGAME.java

/*******************************************
Read a byte - nextByte()
Read a short - nextShort()
Read an int - nextInt()
Read a long - nextLong()
Read a float - nextFloat()
Read a double - nextDouble()
Read a boolean - nextBoolean()
Read a complete line - nextLine()
Read a word - next()
****************************************/
import java.util.*;
import java.lang.*;
import java.io.*;

class Codechef
{
	public static long Sum(long a)
	{
		long  s =0;
			while(a!=0)
		{
			s = s + a%10;
			a = a/10;
		}
		return s;
	}
	public static void main(String[] args) throws java.lang.Exception
	{
            Scanner sc = new Scanner(System.in);
            int t = sc.nextInt();
            while(t>0)
            {
            	int n = sc.nextInt();
            	long  pointc = 0,pointm=0,pc,pm,c,m;
            	while(n>0)
            	{
            		c = sc.nextInt();
            		m = sc.nextInt();
            		pc = Sum(c);
					pm= Sum(m);
            		if(pc == pm)
					{
						pointc++;
						pointm++;
					}
					else if(pc > pm)
						pointc++;
					else
						pointm++;
					
					n--;
            	}
            	
            	if(pointc == pointm)
					System.out.println(2 + " " + pointc);
				else if(pointc > pointm)
					System.out.println(0 + " " + pointc);
				else
					System.out.println(1 + " " + pointm);
				t--;
            }
            
	}
}
